tools/llvm_toolchain.bzl

```diff
@@ -0,0 +1,186 @@
+"""
+Provides a wrapper around `llvm_toolchain` that uses a package provided sysroot
+instead of the system installed libraries.
+"""
+
+load("@bazel_toolchain//toolchain:deps.bzl", "bazel_toolchain_dependencies")
+load(
+    "@bazel_toolchain//toolchain:rules.bzl",
+    _llvm_toolchain = "llvm_toolchain",
+    _llvm_toolchain_files = "llvm",
+)
+
+#  Rename to (hopefully) be less confusing
+llvm_toolchain_dependencies = bazel_toolchain_dependencies
```

Comment:
```suggestion
```

Reply:
We still need to define something here or else do 

```starlark
load(
    "@bazel_toolchain//toolchain:deps.bzl",
    llvm_toolchain_dependencies = "bazel_toolchain_dependencies",
)
```

in the WORKSPACE file